Dockerfile_dist

```diff
@@ -0,0 +1,25 @@
+ARG PYTHON_VERSION=3.11
```

Comment:
Might as well make this 3.12. The library supports it and no reason to not be on the latest.